front/app/api/analyses/__mocks__/useAnalyses.ts

```diff
@@ -0,0 +1,39 @@
+import { IAnalysisData } from '../types';
+
+export const analysesData: IAnalysisData[] = [
+  {
+    id: '1',
+    type: 'analysis',
+    attributes: {
+      created_at: '2021-06-01T08:00:00.000Z',
+      updated_at: '2021-06-01T08:00:00.000Z',
+      participation_method: 'survey',
+    },
+    relationships: {
+      project: {
+        data: {
+          id: '1',
+          type: 'project',
+        },
+      },
+      phase: {
+        data: {
+          id: '1',
+          type: 'phase',
+        },
+      },
```

Comment:
The way it works, there's either a phase, or a project, but never both, so this isn't correct sample data. Probably doesn't really matter in our current tests, but fyi.

I struggle to explain this, but in essence what either of these fields refer to is the place that stores the form definition. For ideation, that's always the project (both for timeline and continuous), for surveys, that's the phase for timeline, the project for continuous.

Reply:
Agree, it's a bit confusing, will fix.